Packages/manifest.json

```diff
@@ -0,0 +1,4 @@
+{
+	"dependencies": {
```

Comment:
Little mysterious changes like this make me nervous. Was this an intentional inclusion in the PR? What process added this blank dependency line?

Reply:
It's for unity's new package manager. I suggest we remove this file from the PR.

Reply:
This wasn't me as well. I can remove it.